Source/WebCore/Modules/mediacontrols/MediaControlsHost.cpp

```diff
@@ -315,6 +315,13 @@ bool MediaControlsHost::inWindowFullscreen() const
     return false;
 }
 
+bool MediaControlsHost::supportsRewind() const
+{
+    if (auto sourceType = this->sourceType())
```

Comment:
This method doesn't exist without `ENABLE(MODERN_MEDIA_CONTROLS)`; fixing in #29293.

Reply:
Thanks!